CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
```

Comment:
Commenting here for threads.

I logged an issue to unbind custom class names added to the output of the generated component incase you wanted to tackle it on this PR or a future PR.

https://github.com/cerner/generator-terra-module/issues/148

Reply:
+1, i made the changes here